### PR TITLE
started decoupling of storage to another package

### PR DESCRIPTION
**Pull request by dharjeezy**
Organize all offchain-storage functions outside of the worker and into a separate file according to #171 and part of #216

### CODE REVIEW

File: dkg-gadget/src/storage/clear.rs [hunk 1]
```diff
@@ -0,0 +1,65 @@
+use crate::{
```
Comment (dutterbutter) [inline dkg-gadget/src/storage/clear.rs:1]: Add license header

File: dkg-gadget/src/messages/public_key_gossip.rs [hunk 2]
```diff
@@ -1,5 +1,7 @@
 // Handles non-dkg messages
```
Comment (dutterbutter) [inline dkg-gadget/src/messages/public_key_gossip.rs:1]: Add license header

File: dkg-gadget/src/messages/misbehaviour_report.rs [hunk 3]
```diff
@@ -1,4 +1,7 @@
-use crate::{types::dkg_topic, worker::DKGWorker, Client};
```
Comment (dutterbutter) [inline dkg-gadget/src/messages/misbehaviour_report.rs:1]: Add license header

**Review by dutterbutter — COMMENTED** [review]
Please add license headers to all files without.

**Review by drewstone — APPROVED** [review]
LGTM!

**Pull request merged**